github/komand_github/actions/add_issue_label/action.py

```diff
@@ -0,0 +1,25 @@
+import komand
+from .schema import AddIssueLabelInput, AddIssueLabelOutput, Input, Output, Component
+# Custom imports below
+
+
+class AddIssueLabel(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='add_issue_label',
+                description=Component.DESCRIPTION,
+                input=AddIssueLabelInput(),
+                output=AddIssueLabelOutput())
+
+    def run(self, params={}):
+        if params.get('organization') and params.get('repository'):
```

Comment:
Whenever you access `params` dict use the I/O constants that are codegenerated, ie `Input.ORGANIZATION`

Reply:
Here's more information. https://komand.github.io/python/sdk.html#actions

Reply:
Fixed. Appreciate the context :)